auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -1550,6 +1584,12 @@ private String createHash(@Nullable String idToken, @Nullable String accessToken
             }
             hash = hash.concat("error=" + error);
         }
+        if (errorDescription != null) {
+            if (!hash.endsWith("&")) {
```

Comment:
this is working, but I don't think it can ever be `false` (except if the values end with a `&` which is wrong).
all these checks should be `endsWith("#")` just in case all previous values where empty/null.